contrib/packs/actions/download.yaml

```diff
@@ -5,23 +5,16 @@
   enabled: true
   entry_point: "pack_mgmt/download.py"
   parameters:
-    packs:
-      type: "array"
-      items:
-        type: "string"
-    repo_url:
+    pack:
```

Comment:
Do we assume StackStorm exchange as the only source of packs? How do run pack install from my local git? Getting it from some random git server is a major use case; and unlike github, we don't even have a concept of org in this case. User will install packs from a mixture of Github/st2exchange, Github/foo, Github/bar, and private git server.